cdap-ui/app/cdap/components/DataPrepHome/index.js

```diff
@@ -150,13 +151,27 @@ export default class DataPrepHome extends Component {
             DataPrepStore.dispatch({
               type: DataPrepActions.disableLoading
             });
+            DataPrepStore.dispatch({
+              type: DataPrepActions.setWorkspaceList,
```

Comment:
would the ordering matter in this case where we disable the loading first before setting the list to an empty array?